src/babel/transformation/transformers/minification/remove-react-prop-types.js

```diff
@@ -0,0 +1,14 @@
+import * as t from "../../../types";
+
+export var metadata = {
+  optional: true,
+  group: "builtin-setup"
+};
+
+export var Property = {
+  exit(node) {
+    if (node.key.name === 'propTypes') {
```

Comment:
@sebmck ok, I'll fix it.
I've also asked in gitter:
 > Hi everyone, I want to add minification option for react prop-types. Could someone help me, how get node parent inside transformation?